client/checkout/upe-styles/index.js

```diff
@@ -21,18 +28,33 @@ const appearanceSelectors = {
 			'woocommerce-invalid',
 			'woocommerce-invalid-required-field',
 		],
+		backgroundSelectors: [
+			'#payment',
+			'#order_review',
+			'form.checkout',
+			'body',
+		],
 	},
 	blocksCheckout: {
 		appendTarget: '#billing.wc-block-components-address-form',
 		upeThemeInputSelector: '#billing-first_name',
-		upeThemeLabelSelector: '.wc-block-components-text-input label',
+		upeThemeLabelSelector:
+			'.wc-block-components-checkout-step__description',
```

Comment:
This computes label styles from the description text (green) instead of the inset label text (red), which stands out better on the page background

![Checkout label style basis](https://cdn-std.droplr.net/files/acc_1104206/E000FU)
_Red old, green new_